.renovaterc.json

```diff
@@ -7,7 +7,7 @@
   },
   "regexManagers": [
     {
-      "fileMatch": ["^tests/functional/fixtures/.env$"],
+      "fileMatch": ["^tests\\/functional\\/fixtures\\/.env$"],
```

Comment:
nit:  Use r-strings

```
"fileMatch": [r"^tests\/functional\/fixtures\/.env$"],
```

Reply:
Thanks @JohnVillalovos. This is a JSON file though, I don't think that's supported?

Reply:
Duh! I was just assuming Python 😟